src/main/java/tempest/GUI/components/GUIComponents.java

```diff
@@ -0,0 +1,26 @@
+package tempest.GUI.components;
+
+import tempest.GUI.Page;
+
+import javax.swing.*;
+
+public class GUIComponents {
+    /**
+     * Returns a clear button
+     *
+     * The page calling this needs a clearInput() method
+     *
+     * Just add this button to the page's panel
+     *
+     * @param page this (instance of class calling getButtonPanel)
+     * @return JButton - A 'Clear' button
+     */
+    public JButton getClearButton(Page page) {
```

Comment:
Can this be refactored to its own file like the other components?

Reply:
I was thinking that this class could be used for components with only 1 method? Else every single reused component will be in a separate class - and pages will have lots of component directory instances

Reply:
Although I get where you're coming from, I think it would make the most logical sense to have every component in a separate class - when you're adding new pages, I don't think that it's obvious to look in that class when most other components will have their own.